### PR TITLE
Crontab minor improvements to testing - master branch

### PR DESCRIPTION
**Pull request by PaulWay**
Updating testing and removing an extraneous debugging print statement.

### CODE REVIEW

File: insights/parsers/crontab.py [hunk 1]
```diff
@@ -147,7 +147,6 @@ def parse_content(self, content):
             cron_match = cron_re.match(line)
             env_match = env_re.match(line)
             if cron_match:
-                print "cron match:", line
                 self.data.append(cron_match.groupdict())
             elif env_match:
                 print "env match:", line
```
Comment (xiangce) [inline insights/parsers/crontab.py]: Here is another `print` you might forget.

Comment (PaulWay) [reply insights/parsers/crontab.py]: I think I fixed that in a recent update...

Comment (bfahr) [reply insights/parsers/crontab.py]: Hey @PaulWay the `print` on line 152 is still there.

**Comment by xiangce** [comment]
@PaulWay - This PR is ready to merge after applying the same updates to the 1.x branch.

**Pull request merged**